pyproject.toml

```diff
@@ -54,6 +54,7 @@ dependencies = [
     "celery>=5.2.7",
     "flower>=2.0.1",
     "maykin-2fa>=1.0.0",
+    "django-setup-configuration>=0.1.0",
```

Comment:
Is this line intentional? It doesn't relate to the issue

Reply:
Yeah, I should have changed the title of the PR. I was making these changes to make it work for objecttypes